apps/aecore/lib/aecore/contract/call.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Aecore.Contract.Call do
+  @moduledoc """
+  Aecore call module implementation.
+  """
+  alias Aecore.Chain.Identifier
+  alias Aecore.Contract.Call
+  alias Aeutil.Parser
+  alias Aeutil.Hash
+
```

Comment:
You should add `use Aecore.Util.Serializable` here probably.

This makes sure you follow behaviour and adds `rlp_encode`, `rlp_decode` functions.